draft-ietf-teep-protocol.md

```diff
@@ -1392,6 +1389,18 @@ The cipher suites defined above do not do encryption at the TEEP layer, but
 permit encryption of the SUIT payload (e.g., using {{I-D.ietf-suit-firmware-encryption}}).
 See {{security}} for more discussion of specific payloads.
 
+For the initial QueryRequest message, unless the TAM has more specific knowledge about the TEEP Agent
+(e.g., if the QueryRequest is sent in response to some underlying transport message that contains a hint),
+the message does not use one of the above cipher suites but instead uses COSE_Sign with multiple signatures,
+one for each algorithm used in any of the cipher suites listed in the supported-cipher-suites
+parameter of the QueryRequest, so that a TEEP Agent supporting any one of them can verify a signature.
+
```

Comment:
Add

"In case that the TAM has already know the cipher suites the TEEP Agent supports, it can include only one signature generated from one of the cipher suite that the TEEP Agent supports in COSE_Sign."

Reply:
added (with marginally different wording)